llms/mlx_lm/tuner/lora.py

```diff
@@ -6,14 +6,27 @@
 
 class LoRALinear(nn.Module):
     @staticmethod
-    def from_linear(linear: nn.Linear, rank: int = 8, scale: float = 20.0):
+    def from_linear(
+        linear: nn.Linear,
+        r: int = 8,
+        lora_alpha: float = 16,
+        lora_dropout: float = 0.05,
+        scale: float = 10.0,
+    ):
         # TODO remove when input_dims and output_dims are attributes
         # on linear and quantized linear
         output_dims, input_dims = linear.weight.shape
+        bias = "bias" in linear
         if isinstance(linear, nn.QuantizedLinear):
             input_dims *= 32 // linear.bits
         lora_lin = LoRALinear(
-            input_dims=input_dims, output_dims=output_dims, rank=rank, scale=scale
+            input_dims=input_dims,
+            output_dims=output_dims,
+            r=r,
+            lora_alpha=lora_alpha,
+            lora_dropout=lora_dropout,
+            scale=scale,
+            bias=bias,
         )
         lora_lin.linear = linear
```

Comment:
You don't really need to pass `bias` here because it will always be overwritten by the next call to set the linear layer.

The `kwarg` is there only so people have the option to instantiate a `LoraLinear` directly (with or without a bias) rather than building from a pre-existing `Linear` layer